Workbooks/SapMonitor2.0/Alerts/Alerts/Alerts.workbook

```diff
@@ -52,7 +52,8 @@
             "name": "ProviderType",
             "type": 1,
             "isRequired": true,
-            "isHiddenWhenLocked": true
+            "isHiddenWhenLocked": true,
+            "value": "SapHana"
```

Comment:
Just a thought
Is this `SapHana` value hard coded here?
Because this same alerts page should work for Net weaver as well right?

Reply:
You are right have updated now